projects/plugins/inspect/composer.json

```
@@ -0,0 +1,56 @@
+{
+	"name": "automattic/jetpack-inspect",
+	"description": "A debugging plugin to inspect Jetpack interactions with WordPress.com.",
+	"type": "wordpress-plugin",
+	"license": "GPL-2.0-or-later",
+	"authors": [
+		{
+			"name": "pyronaur",
+			"email": "hi@pyronaur.com"
+		}
+	],
+	"require": {
+		"automattic/jetpack-autoloader": "@dev",
+		"automattic/jetpack-connection": "@dev",
+		"automattic/jetpack-config": "@dev"
+	},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "1.0.4",
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"config": {
+		"allow-plugins": {
+			"automattic/jetpack-autoloader": true
+		}
+	},
+	"scripts": {
+		"phpunit": [
+			"./vendor/phpunit/phpunit/phpunit --colors=always"
+		],
+		"skip-test-php": "tests/action-skip-test-php.sh",
+		"test-php": [
+			"@composer phpunit"
+		],
+		"build-development": "pnpm run build",
+		"build-production": "pnpm run build-production"
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"mirror-repo": "Automattic/jetpack-inspect",
+		"changelogger": {
+			"link-template": "https://github.com/Automattic/jetpack-inspect/compare/v${old}...v${new}",
+			"versioning": "semver"
+		},
+		"release-branch-prefix": "inspect"
```

Need to configure a few more things here
```suggestion
		"autorelease": true,
		"autotagger": true,
		"mirror-repo": "Automattic/jetpack-inspect",
		"changelogger": {
			"link-template": "https://github.com/Automattic/jetpack-inspect/compare/v${old}...v${new}",
			"versioning": "semver"
		},
		"release-branch-prefix": "inspect",
		"beta-plugin-slug": "jetpack-inspect",
		"wp-svn-autopublish": true
```